acsylla/base.py

```diff
@@ -121,6 +138,35 @@ def bind_dict(self, values: Mapping[str, SupportedType]) -> None:
         a look at the `Statement.bind_dict` function.
         """
 
+    @abstractmethod
+    def set_page_size(self, page_size: int) -> None:
+        """Sets the statement's page size."""
+
+    @abstractmethod
+    def set_page_state(self, page_state: bytes) -> None:
+        """Sets the statement's paging state. This can be used to get the next
+        page of data in a multi-page query.
+
+        Warning: The paging state should not be exposed to or come from
+        untrusted environments. The paging state could be spoofed and potentially
+        used to gain access to other data.
+        """
+
+    @abstractmethod
+    def set_timeout(self, timeout: float) -> None:
+        """Sets the statement's timeout in seconds for waiting for a response from a node.
+        Default: Disabled (use the cluster-level request timeout)"""
+
+    @abstractmethod
+    def set_consistency(self, timeout: float) -> None:
+        """Sets the statement’s consistency level.
+        Default: LOCAL_ONE"""
+
+    @abstractmethod
+    def set_serial_consistency(self, timeout: float) -> None:
+        """Sets the statement’s serial consistency level.
+        Default: Not set"""
```

Comment:
If we finally go for having also this methods at Statment level we should add a minimal test that they are callable, like we have here [1]

[1] https://github.com/acsylla/acsylla/blob/1aae9dde0c4f59982ab32597599523f85643d24e/tests/test_statement.py#L57

Reply:
+